src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -32,6 +32,12 @@ The following can be configured as integration options in `new Replay({})`:
 | networkResponseHeaders  | string[]                 | `[]`     | Additional response headers to capture for URLs defined in `networkDetailAllowUrls`. See [Network Details](#network-details) to learn more.                   |
 | beforeAddRecordingEvent | (event) => event \| null | `i => i` | Filter additional recording events that include console logs and network requests/responses.                                                                  |
 
+## Privacy Configuration
+
+We take privacy seriously, so we provide a number of privacy-oriented settings. Learn more about these in our our <PlatformLink to="/session-replay/privacy/">Session Replay privacy documentation</PlatformLink>.
```

Comment:
Updated, so instead of pointing to `#privacy-configuration` we'll just go to the top of that page.